src/Collections-Support/WideCharacterSet.class.st

```diff
@@ -18,14 +18,19 @@ A byteArrayMap is maintained in parallel with map for fast handling of ByteStrin
 "
 Class {
 	#name : #WideCharacterSet,
-	#superclass : #Collection,
+	#superclass : #CharacterSet,
 	#instVars : [
-		'map',
```

Comment:
super map is already a byteArrayMap...
Having to change class/semantic of a super instance is a smell...

So the change should better be:
- rename super map into byteArrayMap
- rename map into wideCharacterMap
From a bootstrap perspective, this may require several commits though...
(there were live instances to mutate into the Squeak image, so probably so in Pharo!)